website/docs/r/service_v1.html.markdown

```diff
@@ -217,6 +250,15 @@ used in the `request_condition`, `response_condition`, or
 * `priority` - (Optional) A number used to determine the order in which multiple
 conditions execute. Lower numbers execute first. Default `10`.
 
+The `director` block supports:
```

Comment:
Should the director block support all the fields that the director api has available? https://docs.fastly.com/api/config#director

Reply:
That would be ideal but not required (cc/ @hynd).  I would prefer to prioritize the relief of the users for features that can easily be augmented in the future.

Reply:
Good spot - `comment` was missing from the doc, i *think* the only thing actually missing from the code is the `shield` field (which isn't in the underlying go-fastly lib.... yet)

Reply:
Hah, and `capacity` is only half-implemented in go-fastly - i'll PR fixes for both of those first, and will update this accordingly....

Reply:
Thanks @hynd

Reply:
Done - merged upstream and refreshed this PR